### PR TITLE
Add images,HTML and CSS for works section and about me

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
- Created the card section of the project where all my future projects will be displayed
- I have an image, text, tech stack and a button to show the project description
- I created the About section where I talked more about myself and the language I have learned.

>Kindly review and give feedback.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -85,3 +97,154 @@ body {
...
+}
+
+.card-contain li {
+  list-style-type: none;
+}
+
+.about-container {
+  width: 90%;
+  margin: 0 auto;
+  overflow: hidden;
```
Comment (brainconnect93) [inline style.css]: - [ ] I would kindly suggest you stick to the Figma design as much as possible. Currently, the details in your project do not look exactly like what we have in the Figma design.

  - The Image in the project card is not fitting well with the card as we have it in the Figma design
  - The `back end dev` is not aligning properly with what we have in your Figma design

Your design 👇 

rol

Figma design 👇 

card real

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @Roland-Ntwali, 👋 

Good job so far, You really made a great effort working on your project, it looks really awesome! but there are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -85,3 +99,165 @@ body {
...
+.card-contain {
+  display: flex;
+  border: 1px solid #7a869a;
+  width: 88%;
+  margin: 12px auto;
+  border-radius: 16px;
+  background-color: #fff;
+  flex-direction: column;
+  justify-content: center;
+}
```
Comment (didierganthier) [inline style.css]: Hey 👋🏽 keep going, I see that you tried to implement the changes requested by the previous reviewer 🚀 although I'm pretty sure you made valuable progress we're not there yet. Look to adjust your design to the Figma by respecting the correct dimensions. See below 👇🏽 

### Your design:
Screenshot 2023-01-13 at 07 16 46

### Figma Design
Screenshot 2023-01-13 at 07 17 35

There are several ways to fix this by playing with the margins and the padding, you got this 💪🏽

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @Roland-Ntwali ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Roland-Ntwali,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**